src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_rate.scss

```diff
@@ -166,6 +236,8 @@ the `is-disabled` state.
     }
 
     .rate-difference {
+        vertical-align: top;
```

Comment:
This does not seem to make a difference. I would also prefer baseline alignment with the buttons if that is possible.